type-definitions/index.d.ts

```diff
@@ -19,6 +19,8 @@ declare interface DisposeFn {
   (): void | Promise<any>;
 }
 
-export default function create<A>(f: (add: (a: A) => void,
+declare function create<A>(f: (add: (a: A) => void,
                                       end: (x?: A) => void,
                                       error: (e: Error) => void) => void | DisposeFn ): Stream<A>;
+
+export = create;
```

Comment:
i assume that might work. but still feels hacky.
version 1.1.3 will compile successfully on default import but the module will output undefined.
issue is probably coming from #4 after migrating into a rollup & buble.
1.0.1 works good for me.